app/views/admin/contacts/_contacts.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render(contacts, contactable: contactable) %>
+
+<% contacts.each do |contact| %>
+  <% contact.non_english_localised_models([:contact_numbers]).each do |translated_contact| %>
+    <div class="govuk-body govuk-!-margin-top-8">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: title,
+      margin_bottom: 4,
+    } %>
+    </div>
+    <%= render "translated_contact", translated_contact: translated_contact, contactable: contactable %>
```

Comment:
This is only ever rendering one translation at the moment as it only outputs the last thing returned to the each. This is when i've translated 4 offices 

<img width="478" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/d5a53a0c-9157-49a3-96c0-66d7e9aa5876">


If you do 

```
    <%= concat(render "translated_contact", translated_contact: translated_contact, contactable: contactable) %>

```
it works as it concats the output  to the above html.